### PR TITLE
Removed Wear OS from short_description

### LINKED ISSUE #660: short_descriptions too long
Fastlane's `short_description.txt` are limited to 80 chars – everything beyond that will simply be truncated. Seems like all of yours are too long:

```
de-DE/short_description.txt has 88 chars
pl-PL/short_description.txt has 94 chars
pt-BR/short_description.txt has 93 chars
ru-RU/short_description.txt has 135 chars
sk-SK/short_description.txt has 115 chars
uk-UA/short_description.txt has 141 chars
```

Just to let you know so you can take care :wink:

### PR DESCRIPTION
**Pull request by MagicLike**
Resolves #660

### CODE REVIEW

File: fastlane/metadata/android/en-US/short_description.txt [hunk 1]
```diff
@@ -1 +1 @@
-Free and open-source 2FA app with encrypted backups and Wear OS support
```
Comment (IzzySoft) [inline fastlane/metadata/android/en-US/short_description.txt:1]: Actually, for `en-US` this is OK (71 byte altogether) and thus not required to be shortened. As pointed out in #660 it's rather `de-DE`, `pl-PL`, `pt-BR`, `ru-RU`, `sk-SK` and `uk-UA` which need to be shortened.

**Review by IzzySoft — COMMENTED** [review]
Should be fine if `en-US` is used as base for some translation service (Weblate, Transifex, whatever). Else shortening `en-US` is not needed, as pointed out.

File: fastlane/metadata/android/en-US/short_description.txt (hunk 1 above)
Comment (MagicLike) [reply fastlane/metadata/android/en-US/short_description.txt:1]: https://github.com/jamie-mh/AuthenticatorPro/issues/660#issuecomment-1453921055

**Comment by MagicLike** [comment]
> Should be fine if `en-US` is used as base for some translation service (Weblate, Transifex, whatever). Else shortening `en-US` is not needed, as pointed out.

https://github.com/jamie-mh/AuthenticatorPro/issues/660#issuecomment-1453921055

**Comment by IzzySoft** [comment]
LGTM then, and thanks!

**Comment by jamie-mh** [comment]
Thanks!

**Pull request merged**